test/FullRange.t.sol

```diff
@@ -80,15 +77,17 @@ contract TestFullRange is Test, Deployers, GasSnapshot {
     PoolKey keyWithLiq;
     PoolId idWithLiq;
 
-    PoolModifyPositionTest modifyPositionRouter;
-    PoolSwapTest swapRouter;
-
     function setUp() public {
-        token0 = new MockERC20("TestA", "A", 18, 2 ** 128);
-        token1 = new MockERC20("TestB", "B", 18, 2 ** 128);
-        token2 = new MockERC20("TestC", "C", 18, 2 ** 128);
+        deployFreshManagerAndRouters();
+        router = new HookEnabledSwapRouter(manager);
+        (currency0, currency1) = deployMintAndApprove2Currencies();
```

Comment:
these seem to be unused, and then instead token0 token1 are deployed below?
I'd rather remove the `tokenX` variables and use global `currencyX` for consistency

Reply:
changing to use `deployTokens` and remove unused. The currencyX are annoying here because:
- all the tests are written around MockERC20 and dont want to have to do Currency.unwrap everywhere
- we have 3, not just 2 so can't use the globals for all :(